x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/action/EsqlCapabilities.java

```diff
@@ -843,7 +843,7 @@ public enum Cap {
         /**
          * Support change point detection "CHANGE_POINT".
          */
-        CHANGE_POINT(Build.current().isSnapshot()),
+        CHANGE_POINT,
```

Comment:
If the intention is to have it in non-snapshot releases, I think we also have to change the [grammar](https://github.com/elastic/elasticsearch/blob/main/x-pack/plugin/esql/src/main/antlr/EsqlBaseParser.g4#L61)